src/app/Analytics/LocalMonitoring/RecommendationsForLocalMonitoring/RemoteMonitoring/RecommendationTables.tsx

```diff
@@ -96,11 +109,67 @@ const RecommendationTables = (props: { endTimeArray; setEndTimeArray; SREdata; s
         setCurrentData(current_arr);
         setRecommendedData(recommended_arr);
         setChartDetailsData(chartDetailsObject);
-      }
-    };
+          // setBoxPlotData(boxPlot_arr);
```

Comment:
Do we need this? if not, remove the unwanted commented out code.

Reply:
these r debugging statements, in the other pr these r mostly removed